templates/vue/src/components/tyde/activities/CircleOfSupport/index.vue

```diff
@@ -3,13 +3,24 @@
 </template>
 
 <script>
+import moment from "moment-timezone"
+import client from "@/services/TapestryAPI"
+
 export default {
   data() {
     return {
-      circles: [],
-      communities: [],
-      connections: [],
+      cos: {
+        id: "",
+        circles: [],
+        communities: [],
+        connections: [],
+        members: {},
+        timestamp: moment(),
+      },
     }
   },
+  async mounted() {
+    this.cos = await client.getCosActivity()
```

Comment:
This would just return null if there is no existing CoS saved already. Maybe you want to create a base case here (e.g. `if (retVal !== null)...`) otherwise your data variable would always get overwritten.